buildout.cfg

```
@@ -23,23 +25,32 @@ eggs =
     Pillow
     recensio.plone [test]
 
-
 [createcoverage]
 recipe = zc.recipe.egg
 eggs = createcoverage
 
+[solr]
+recipe = kitconcept.recipe.solr
+src = https://www.apache.org/dyn/closer.lua/solr/solr/9.5.0/solr-9.5.0.tgz?action=download
```

Is there a reason why we are not using the latest version?
```suggestion
src = https://www.apache.org/dyn/closer.lua/solr/solr/9.9.0/solr-9.9.0.tgz?action=download
```